includes/Core/Consent_Mode/REST_Consent_Mode_Controller.php

```diff
@@ -163,8 +164,9 @@ protected function get_rest_routes() {
 								if ( ! function_exists( 'get_plugins' ) ) {
 									require_once ABSPATH . 'wp-admin/includes/plugin.php';
 								}
-								foreach ( array_keys( get_plugins() ) as $installed_plugin ) {
-									if ( $installed_plugin === $plugin ) {
+								foreach ( get_plugins() as $plugin_file => $installed_plugin ) {
```

Comment:
Before we go through the loop, let's check using the default `$plugin` as this is the most likely case. This way, if the `PluginURI` changes in the future, it won't break the detection in the majority of cases.

Reply:
Updated, thank you!